.docker/local-dev.env

```diff
@@ -0,0 +1,13 @@
+# Disable XRay locally
+AWS_XRAY_SDK_ENABLED=false
+
+AWS_DYNAMODB_ENDPOINT_URL=https://dynamodb.us-east-1.amazonaws.com
```

Comment:
Do you need to set `AWS_PROFILE=pocket-dev-PocketSSODataLearning` in this file?

Reply:
I set the environment variable in the run configuration (line 8):
```xml
<env name="AWS_PROFILE" value="pocket-dev-PocketSSODataLearning" />
```
Not really sure why I chose to do that. Either way would be fine with me.